tests/functional/monitoring/conftest.py

```diff
@@ -905,8 +905,17 @@ def stop_rgw():
         nonlocal rgw_deployments
         for rgw_deployment in rgw_deployments:
             rgw = rgw_deployment["metadata"]["name"]
-            logger.info(f"Downscaling deployment {rgw} to 0")
-            oc.exec_oc_cmd(f"scale --replicas=0 deployment/{rgw}")
+            logger.info(
+                f"Updating readiness probe of deployment {rgw} to make rgw pod down"
+            )
+            cmd = (
```

Comment:
The change in readiness probe should be restored when testing is done.

Reply:
At the end deleting the deployment rgw and operator pod restarts it with the right readiness probe condition. This has been verified.